css/common/button/button.less

```
@@ -0,0 +1,17 @@
+.button {
+    line-height: 20px;
+    background-color: #0F8FEE;
+    border-radius: 1px;
+    color: white;
+    padding: 9px 15px;
+    border: none;
+    text-align: center;
+
+    &:hover {
```

```suggestion
    &:hover, &:active {
```
можно объединять селекторы с одинаковыми свойствами

---

исправил